apps/codeforafrica/src/components/FeaturedArticle/FeaturedArticle.js

```diff
@@ -0,0 +1,161 @@
+import { Link } from "@commons-ui/next";
+import Button from "@mui/material/Button";
+import CardActionArea from "@mui/material/CardActionArea";
+import Grid from "@mui/material/Grid";
+import Typography from "@mui/material/Typography";
+import React from "react";
+
+import ArticleCard from "@/codeforafrica/components/ArticleCard";
+import ArticleCardContent from "@/codeforafrica/components/ArticleCardContent";
+import ArticleCardMedia from "@/codeforafrica/components/ArticleCardMedia";
+
+const ArticleCardList = React.forwardRef(function ArticleCardList(props, ref) {
+  const {
+    date,
+    href,
+    src,
+    summary,
+    title,
+    variant = "standard",
+    ...other
+  } = props;
+
+  return (
+    <ArticleCard
+      sx={{
+        width: "100%",
+        height: "100%",
+      }}
+      {...other}
+      ref={ref}
+    >
+      <CardActionArea
+        component={Link}
+        sx={{ position: "relative" }}
+        href={href}
+      >
+        <Grid
+          container
+          direction={{ xs: "column", md: "row" }}
+          justifyContent="space-between"
+          wrap="nowrap"
+        >
+          <Grid item xs={12} md={variant === "cover" ? 12 : "auto"}>
+            <ArticleCardMedia
```

Comment:
Provide an empty alternative text for the image, currently there isn't [any](https://github.com/CodeForAfrica/ui/blob/main/apps/codeforafrica/src/components/ArticleCardMedia/ArticleCardMedia.js#L11)